COMPILING.txt

```diff
@@ -49,7 +42,7 @@ If you're having trouble compiling, please ask in #pioneer on
 irc.freenode.net.
```

Comment:
While you're at it, you could add "(most users are in CET zone)", even if it's not true for you. Not uncommon to see in my backlog people who enter #pioneer at 2 AM asking for help.